task3.py

```diff
@@ -0,0 +1,13 @@
+""" Task 3"""
+
+
+def all_numb_sum(n):
+    """
+    сумму всех чисел до заданного
+    """
+    if n == 1:
+        return 1
+    return n + all_numb_sum(n - 1)
```

Comment:
Interesting solution, but not sure that it’s effective
```suggestion
def all_numb_sum(n):
    return n * (n + 1) // 2
```